flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/generated/ProcessTableRunner.java

```diff
@@ -183,34 +184,53 @@ private void processMethod(RunnableWithException method) throws Exception {
         }
     }
 
+    @SuppressWarnings("unchecked")
     private void moveStateToFunction() throws IOException {
         Arrays.fill(stateCleared, false);
         for (int i = 0; i < stateHandles.length; i++) {
-            final RowData value = stateHandles[i].value();
-            stateToFunction[i] = value;
+            final State stateHandle = stateHandles[i];
+            if (!(stateHandle instanceof ValueState)) {
```

Comment:
nit: the code would be cleaner with the if as positive and without the continue: 
```
 if ((stateHandle instanceof ValueState)) {
           final ValueState<RowData> valueState = (ValueState<RowData>) stateHandle;
           final RowData value = valueState.value();
           valueStateToFunction[i] = value;
  }
```

Reply:
I'm following the [early return principle](https://dev.to/shameel/-clean-coding-tip-early-return-principle-47kj) here. I think in the end this is a matter of taste. Feel free to code it differently in your PRs.